pkg/ebpf/gadgets/http/tracer/bpf/http-sniffer.c

```diff
@@ -0,0 +1,623 @@
+#include "sniffer_strcuts.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+typedef unsigned short int sa_family_t;
```

Comment:
Put it in the header file and add a comment why you define this type and where it is coming from. Do it for all the types you define.

Reply:
Deleted it